### PR TITLE
1.0.1 update

### PR DESCRIPTION
**Pull request by samuelmi2**
1.0.1 update

### CODE REVIEW

File: teardown.sh [hunk 1]
```diff
@@ -81,8 +81,8 @@ if directory_exists "${ISTIO_DIR}" ; then
   done
 fi
 
-#### delete local istio binary
-rm -rf "${ISTIO_DIR}"
+# #### delete local istio binary
+# rm -rf "${ISTIO_DIR}"
```
Comment (chrislovecnm) [inline teardown.sh]: I think we need to remove this comment.

File: setup.sh [hunk 2]
```diff
@@ -25,17 +25,15 @@ set -e
...
-
-  # Source utility functions for checking the existence of various resources.
-  source "${SHARED_DIR}/verify-functions.sh"
-else
-  echo "${SHARED_DIR} does not exist, please check the variable settings in \
-the properties file."
-  echo "Also make sure you clone the shared repository located at: \
-https://github.com/GoogleCloudPlatform/gke-istio-shared"
+if ! directory_exists "${SHARED_DIR}" ; then
+  echo "${SHARED_DIR} does not exist, please check the variable "
```
Comment (chrislovecnm) [inline setup.sh]: How about we tell them something about what the shared directory is??  What is a better error message?

**Comment by samuelmi2** [comment]
@chrislovecnm I've made the changes in the latest push

File: teardown.sh (hunk 1 above)
Comment (samuelmi2) [reply teardown.sh]: removed.

File: setup.sh (hunk 2 above)
Comment (samuelmi2) [reply setup.sh]: the new language is: 
```
  echo "${SHARED_DIR} which contains shared istio scripts does "
  echo "not exist, please check the variable settings in the properties "
  echo "file. Exiting..."
```

**Comment by chrislovecnm** [comment]
/lgtm

Squash and I will merge

**Pull request closed without merging**